.tests/alexa.sh

```diff
@@ -33,14 +38,14 @@ check_rank () {
     fi
 
     # Check if the rank is at or below 200K
-		if [ "$alexa_rank" -gt "$max_ranking" ]; then
```

Comment:
I was just about to comment that you've used tabs instead of spaces which is against the editorconfig settings but then I realized that I'm the one who used it... 🙄 *cough* *cough*